storage/statedb/cache_redis.go

```diff
@@ -115,3 +115,7 @@ func (cache *RedisCache) SubscriptionChannel(channel string) <-chan *redis.Messa
 func (cache *RedisCache) UpdateStats() interface{} {
 	return nil
 }
+
+func (cache *RedisCache) SaveToFile(filePath string, concurrency int) error {
+	return nil
```

Comment:
Logging or returning an error might be needed?

Reply:
@winnie-byun This is only to meet the interface of cache, do you think an error would be needed for this case?

Reply:
When using redis cache and `SaveTrieNodeCacheToDisk` is called, one will not see any error log and think it is successfully stored to file.